src/views/VenueDetails/BookVenue/index.jsx

```diff
@@ -1,12 +1,102 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import styles from './styles.module.css'
+import { Calendar } from './Calendar'
+import CircularProgress from '@mui/material/CircularProgress'
+import PhoneIcon from '@mui/icons-material/Phone'
+import MailIcon from '@mui/icons-material/Mail'
+import { FaFacebookF } from 'react-icons/fa'
+import { linkPaths } from '../../../LinksPaths'
+import { AiFillInstagram, AiOutlineTwitter } from 'react-icons/ai'
+import { GrLanguage } from 'react-icons/gr'
+
+const dayInMilliseconds = 24 * 60 * 60 * 1000
```

Comment:
`const DAY_IN_MILISECONDS = …`
The thing with the const that are valuated once per file (in the compilation process) is that you should use upper case with `_` sign. That's the standard